CHANGELOG.md

```diff
@@ -90,6 +90,7 @@
 * Removed deprecated and obsoleted `EventsManager` typealias, `StatusView.delegate`, `StatusView.canChangeValue`, `RouteLegProgress.upComingStep`, `StatusViewDelegate`, `DeprecatedStatusViewDelegate` and `BottomBannerViewController.init(delegate:)`. ([#2993](https://github.com/mapbox/mapbox-navigation-ios/pull/2993))
 * Fixed a potential memory leak when using `MultiplexedSpeechSynthesizer`. ([#3005](https://github.com/mapbox/mapbox-navigation-ios/pull/3005))
 * Fixed an issue where instruction banners could appear in the wrong color after switching between `Style`s. ([#2977](https://github.com/mapbox/mapbox-navigation-ios/pull/2977))
+* Improved the road lable accuracy during turn-by-turn navigation by using the e-horizon notification road name to improve the road feature result. ([#3019](https://github.com/mapbox/mapbox-navigation-ios/pull/3019))
```

Comment:
```suggestion
* Improved the road name labeling accuracy during turn-by-turn navigation by using `Notification.Name.electronicHorizonDidUpdatePosition`. ([#3019](https://github.com/mapbox/mapbox-navigation-ios/pull/3019))
```

Reply:
As a second thought: I'm not even sure that we should add this part, since no public APIs are affected.

Reply:
Thanks for the suggestion, I'll remove this part.